common/modules/gcp-stackdriver/notification_alerts_slack.tf

```diff
@@ -0,0 +1,12 @@
+resource "google_monitoring_notification_channel" "alerts_slack" {
```

Comment:
@amatas Did you try applying this? What manual steps are needed to finish configuration on Slack side? Do we need to update this README entry?
https://github.com/gpii-ops/gpii-infra/tree/master/gcp#to-receive-notifications-in-slack

Reply:
I can not apply this PR without a possible break of the actual resources. I will address any issue as the PR goes through the pipeline.

Reply:
Ok, I think we figured out what needs to be done to set up Slack integration properly during today's session.